llvm/lib/IR/ConstantFold.cpp

```diff
@@ -729,13 +729,15 @@ Constant *llvm::ConstantFoldBinaryInstruction(unsigned Opcode, Constant *C1,
   // Neither constant should be UndefValue, unless these are vector constants.
   assert((!HasScalarUndefOrScalableVectorUndef) && "Unexpected UndefValue");
 
+  Constant *Absorber = ConstantExpr::getBinOpAbsorber(
+      Opcode, C1->getType(), /*AllowLHSAbsorber*/ true);
```

Comment:
```suggestion
      Opcode, C1->getType(), /*AllowLHSConstant*/ true);
```
To match the parameter name.

Reply:
Okay.